src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/InputParameterNameWhitespaceInAccessModifierProperty.java

```diff
@@ -0,0 +1,23 @@
+/*
+ParameterName
+format = ^h$
+ignoreOverridden = (default)false
+accessModifiers = \tpublic
```

Comment:
@rnveach

Reply:
> config like this: public\t,\t,

Reply:
In other PR, we had `public, \t`, case taken from this PR is with tab character "on" the access modifier. The first trim in both methods took care of the single `\t`, the second trim takes care of the `\t` "on" the access modifier.